wagtail/contrib/search_promotions/templates/wagtailsearchpromotions/search_promotion_column.html

```diff
@@ -0,0 +1,15 @@
+{% load i18n wagtailadmin_tags %}
+<td>
+    {% for searchpick in value %}
+        {% if searchpick.page %}
+            <a href="{% url 'wagtailadmin_pages:edit' searchpick.page.id %}" class="nolink">{{ searchpick.title }}</a>
+        {% else %}
+            <a href="{{ searchpick.external_link_url }}" class="nolink" target="_blank" rel="noreferrer">
+                {{ searchpick.title }} {% icon classname="initial" name="link-external" title=_("External link") %}
+            </a>
+        {% endif %}
+        {% if not forloop.last %}, {% endif %}
+    {% empty %}
+        {% trans "None" %}
+    {% endfor %}
+</td>
```

Comment:
Needs a newline at end of file

```suggestion
</td>

```